src/index.html

```diff
@@ -6,9 +6,44 @@
         content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <title>Moyo header</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:500&display=swap" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>Moyo header</h1>
+
+<header class="header">
+  <div class="logo"><a href="#"><img src="images/logo.png" alt="logo"></a></div>
```

Comment:
One tag - one line. Use new lines for nested tags. Don't write nested tags in one line.